src/m365/spo/commands/listitem/listitem-get.spec.ts

```diff
@@ -220,6 +250,22 @@ describe(commands.LISTITEM_GET, () => {
     assert.strictEqual(actualId, expectedId);
   });
 
+  it('returns listItemInstance object by uniqueId when list item is requested', async () => {
+    sinon.stub(request, 'get').callsFake(getFakes);
+
+    command.allowUnknownOptions();
+
+    const options: any = {
+      debug: true,
+      listTitle: 'Demo List',
+      webUrl: webUrl,
+      uniqueId: expectedUniqueId
+    };
+
+    await command.action(logger, { options: options } as any);
+    assert.strictEqual(actualUniqueId, expectedUniqueId);
```

Comment:
I don't think this whole setup with `actualUniqueId` is a good thing to use. For one thing it's set the first time getFakes is run, but not cleared after every test case, potentially leading to unclear testresults.
I'd rather we test that the logger has logged a specific response.